src/common/components/organisms/MobileNavbar.tsx

```
@@ -141,60 +156,60 @@ const MobileNavbar: React.FC<MobileNavbarProps> = ({
   const getTabLabel = useCallback((tab: TabItem, index: number): string => {
     // If tab already has a label, use it
     if (tab.label) return tab.label;
-    
+
     // Special consolidated views
     if (tab.id === 'consolidated-media') return "Media";
     if (tab.id === 'consolidated-pinned') return "Pinned";
-    
+
     // Use custom tab name from SpaceConfig if available
     if (tabNames && tabNames[index]) return tabNames[index];
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)